src/main/java/tennisclub/dao/LessonDaoImpl.java

```diff
@@ -51,9 +51,16 @@ public Lesson findById(Long id) {
     }*/
 
     @Override
-    public List<Lesson> findByCapacity(int capacity) {
+    public List<Lesson> findByCapacity(Integer capacity) {
```

Comment:
You are right, the wrapper classes are used mostly because of the need for nullability - in the case of argument of type `Integer`, I am not sure, whether `null` could be actually passed as an argument 😰

Reply:
However, as pointed out by Ondřej, we can actually consider a capacity, which can be actually `null` (unlimited capacity), so this could be considered as resolved by changing it to a `nullable` attribute 😊